### PR TITLE
Prepare v0.67.0 release

### PR DESCRIPTION
**Pull request by ashutosh-narkar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,13 +34,34 @@ server:
...
 
-### Breaking Changes
+### Runtime, Tooling, SDK
+- repl: Add support for correctly loading bundle modules ([#6872](https://github.com/open-policy-agent/opa/issues/6872)) authored by @ashutosh-narkar
+- plugins/discovery: Allow un-registration of discovery listener ([#6851](https://github.com/open-policy-agent/opa/pull/6851)) authored by @mjungsbluth. The discovery plugin allows to register a bundle download status listener but previously did not offer a method to unregister that listener
+- plugins/logs: Reduce amount of work performed inside global lock in decision log plugin ([#6859](https://github.com/open-policy-agent/opa/pull/6859)) authored by @johanfylling
+- plugins/rest: Add a new client credential attribute to support Azure Workload Identity. This would allow workloads deployed on an Azure Kubernetes Services (AKS) cluster to authenticate and access Azure cloud resources ([#6802](https://github.com/open-policy-agent/opa/pull/6802)) authored by @ledbutter
+- cmd/inspect: Add ability for opa inspect to inspect a single file outside off any bundle ([#6873](https://github.com/open-policy-agent/opa/pull/6873)) authored by @tjons
```
Comment (philipaconrad) [inline CHANGELOG.md:48]:
```suggestion
- cmd/inspect: Add ability for opa inspect to inspect a single file outside of any bundle ([#6873](https://github.com/open-policy-agent/opa/pull/6873)) authored by @tjons
```
[typo]

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,13 +34,34 @@ server:
...
+- format: Produce error when `--rego-v1`  formatted module has rule name conflicting with keyword ([#6833](https://github.com/open-policy-agent/opa/issues/6833)) authored by @johanfylling
+- topdown: Add cap to caches for regex and glob built-in functions ([#6828](https://github.com/open-policy-agent/opa/issues/6828)) authored by @johanfylling. This fixes possible memory leaks where caches grow uncontrollably when large amounts of regexes or globs are generated or originate from the input document.
 
-### Breaking Changes
+### Runtime, Tooling, SDK
+- repl: Add support for correctly loading bundle modules ([#6872](https://github.com/open-policy-agent/opa/issues/6872)) authored by @ashutosh-narkar
+- plugins/discovery: Allow un-registration of discovery listener ([#6851](https://github.com/open-policy-agent/opa/pull/6851)) authored by @mjungsbluth. The discovery plugin allows to register a bundle download status listener but previously did not offer a method to unregister that listener
```
Comment (philipaconrad) [inline CHANGELOG.md:45]:
```suggestion
- plugins/discovery: Allow un-registration of discovery listener ([#6851](https://github.com/open-policy-agent/opa/pull/6851)) authored by @mjungsbluth. The discovery plugin allows OPA to register a bundle download status listener but previously did not offer a method to unregister that listener
```
[suggestion, missing word]

**Review by philipaconrad — APPROVED** [review]
Some minor typos spotted-- other than that, I think we're good to cut the release! :smile:

Thanks for wrangling the release notes, @ashutosh-narkar!

File: CHANGELOG.md (hunk 1 above)
Comment (ashutosh-narkar) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (ashutosh-narkar) [reply CHANGELOG.md]: Done

**Review by philipaconrad — APPROVED** [review]

**Pull request merged**